packages/ckeditor5-dev-build-tools/src/plugins/cssStyles.ts

```diff
@@ -0,0 +1,305 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import { createFilter } from '@rollup/pluginutils';
+import { parse, type Rule, type Declaration, type Stylesheet } from 'css';
+import type { Plugin, GetModuleInfo, OutputBundle, OutputChunk, NormalizedOutputOptions } from 'rollup';
+
+/**
+ * Filter files only with `css` extension.
+ */
+const filter = createFilter( [ '**/*.css' ] );
+
+export function cssStyles(): Plugin {
```

Comment:
The name of this plugin could a little bit more descriptive. What do you think about `splitCss` or something similar?